src/ui/imageballoonpanelview.js

```diff
@@ -86,6 +83,16 @@ export default class ImageBalloonPanelView extends BalloonPanelView {
 	}
 
 	/**
+	 * @inheritDoc
+	 */
+	init() {
+		super.init();
```

Comment:
PS. I wouldn't require that if `BalloonPanelView` was from this package. But it's a separate package so it can change independently.